src/ucp/core/ucp_context.c

```diff
@@ -295,6 +295,14 @@ static ucs_config_field_t ucp_config_table[] = {
    "Experimental: enable new protocol selection logic",
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
+  {"KEEPALIVE_TIMEOUT", "0us",
+   "Time period to initiate keepalive operation (0 - disabled).",
+   ucs_offsetof(ucp_config_t, ctx.keepalive_timeout), UCS_CONFIG_TYPE_TIME},
+
+  {"KEEPALIVE_NUM_EPS", "0",
+   "Number of endpoints to test for keepalive per round (0 - disabled,\n"
+   "inf - check all EPS",
```

Comment:
Maximal number of endpoints to check on every keepalive round (0 - disabled, inf - check all endpoints on every round)